snd-server/services/config.py

```diff
@@ -10,7 +10,11 @@ class Config:
     spotify_client_id: str = os.environ["SND_SPOTIFY_CLIENT_ID"]
     spotify_client_secret: str = os.environ["SND_SPOTIFY_CLIENT_SECRET"]
 
-    sounds_storage_path = "dataset.csv"
-    artists_storage_path = "artists.csv"
+    sounds_storage_path = "data/dataset.csv"
+    transformed_sounds_storage_path = "data/transformed_dataset.csv"
+    artists_storage_path = "data/artists.csv"
+    scaler_storage_path = "data/sc.joblib"
+
+    archive_storage_path = "snd-server/archive.zip"
```

Comment:
```suggestion
    archive_storage_path: str = os.environ["SND_RECOMMENDER_ARCHIVE_PATH"]
```

Reply:
Or if we're going to commit this file into repo for now, then it doesn't have to be env variable, but make sure you read the file correctly (perhaps relative to the project root path?) 
Because when the server is launched in prod, active directory could be different